### PR TITLE
Added display page for peer advice page; working on displaying form i…

### PR DESCRIPTION
**Pull request by rokiasissoko**
Display page for form input here.

### CODE REVIEW

**Review by DianaEsp — COMMENTED** [review]
I think everything looks good! Can't wait to see it come together.

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<dependency>
```
Comment (tucosh) [inline pom.xml]: I'm not sure this partial pom.xml is useful, until the file is complete enough to compile. Mathew has another pr to do that. You'll probably need to merge after that's checked in. In the meantime, I'm not really sure how I can validate whether the rest of this works, other than just eyeballing it

Comment (rokiasissoko) [reply pom.xml]: This pom.xml isn't useful at all. I will merge with Matthew's pom.xml.

File: src/main/webapp/peer_advice.html [hunk 2]
```diff
@@ -3,33 +3,43 @@
...
-                <li><a href="#">Health</a></li>            
-                <li><a href="#">Events</a></li>
-            </ul>
-        </nav>
-    </div> 
+<body class="w3-light-grey">
+    <div class="w3-top">
+        <div class="w3-bar w3-white w3-padding w3-card" style="letter-spacing:4px;">
+            <a href="Home.html" class="w3-bar-item w3-button w3-padding-large w3-white">Home</a>
+            <a href="peer-advice.html" class="w3-bar-item w3-button w3-hide-small w3-padding-large w3-hover-white">Peer Advice</a>
```
Comment (nnooney) [inline src/main/webapp/peer_advice.html:30]: I believe this should be "peer_advice", not "peer-advice". Also, you should update the navigation bar on other HTML pages to link to this new page.

**Review by nnooney — CHANGES_REQUESTED** [review]
Just one comment to help with the site navigation! Looks great otherwise!

File: src/main/webapp/peer_advice.html (hunk 2 above)
Comment (rokiasissoko) [reply src/main/webapp/peer_advice.html:30]: Moved all content of "peer_advice.html" to DIana's "Peer.html". Form can be found at the end of the page

**Pull request closed without merging**